src/lib/components/VideoCard/VideoCard.scss

```diff
@@ -1,7 +1,3 @@
-@use "uswds-core" as * with (
-  $theme-show-notifications: false
-);
```

Comment:
I think this was leftover from previous work that utilized USWDS mixins. Should no longer be required.